src/components/grid/index.jsx

```diff
@@ -23,6 +21,10 @@ const Grid = ({ gridNodes, onSelectNode, highlightColor }) => {
       <td
         key={`${row}-${col}`}
         onClick={() => onSelectNode(row, col)}
+        onMouseEnter={() => {
+          if(isLongPress) 
+            onSelectNode(row, col)
```

Comment:
Is there a cleaner way of doing this?

Reply:
Looks clean enough to me. The main thing is readability by other developers.